test/martha/martha_v3.test.js

```diff
@@ -160,25 +91,25 @@ test.serial.afterEach(() => {
 });
 
 test.serial('martha_v3 resolves a valid DOS-style url', async (t) => {
-    getJsonFromApiStub.onFirstCall().resolves(dataObjectServiceObject);
+    getJsonFromApiStub.onFirstCall().resolves(sampleDosResponse);
     const response = mockResponse();
     await marthaV3(mockRequest({ body: { 'url': 'dos://abc/123' } }), response);
     const result = response.send.lastCall.args[0];
-    t.deepEqual(Object.assign({}, result), fullExpectedResult(googleSAKeyObject));
+    t.deepEqual(Object.assign({}, result), sampleDosMarthaResult(googleSAKeyObject));
     t.is(response.statusCode, 200);
 });
 
 test.serial('martha_v3 resolves a valid DRS-style url', async (t) => {
-    getJsonFromApiStub.onFirstCall().resolves(dataObjectServiceObject);
+    getJsonFromApiStub.onFirstCall().resolves(sampleDosResponse);
```

Comment:
This prior comment got lost in a code change: Is this really correct that a DRS url resolves to DOS? If so, maybe add a comment to this test explaining why this odd result is correct?

Reply:
I'm not sure, this particular test is pre-existing (I just renamed the variable)

Will find out & fix if appropriate